modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/MediaDescriptors/Extensions/MediaDescriptorExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.IO;
+using System.Linq;
+
+namespace Volo.CmsKit.MediaDescriptors.Extensions
+{
+    public static class MediaDescriptorExtensions
+    {
+        public static bool IsValidMediaFileName(this string name)
+        {
+            if (string.IsNullOrWhiteSpace(name))
+            {
+                return false;
+            }
+            
+            // "con" is not valid folder/file name for Windows OS
```

Comment:
Isn't this provided by a class in System.IO?

Reply:
There is no predefined method for that.